### PR TITLE
Improved Expect Revert to avoid false positive

### PR DESCRIPTION
**Pull request by jjtechuy**
Improved Expect Revert to avoid false positive

### CODE REVIEW

**Comment by jjtechuy** [comment]
@leoiovlabs, please review, approve and merge. Thanks!

File: results/test-results.xml [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    <testcase name="Contract: RRC20 Events: Should fire Transfer event normally on a zero transfer" time="0.0540" classname="Events: Should fire Transfer event normally on a zero transfer">
+    </testcase>
+    <testcase name="Contract: RRC20 Events: Should fire Approval event properly" time="0.0190" classname="Events: Should fire Approval event properly">
+    </testcase>
+  </testsuite>
+  <testsuite name="Contract: RRC20Factory" timestamp="2021-09-20T14:05:52" tests="1" file="/Users/jjtechera/Documents/GitHub/rskj-truffle-tests/test/token/rrc20/rrc20Factory.js" time="0.1270" failures="0">
+    <testcase name="Contract: RRC20Factory Verify a Human Standard Token once deployed using both verification functions." time="0.1270" classname="Verify a Human Standard Token once deployed using both verification functions.">
+    </testcase>
+  </testsuite>
+</testsuites>
```
Comment (leoiovlabs) [inline results/test-results.xml:255]: `test-results.xml` should not be uploaded to the repo.

**Review by leoiovlabs — CHANGES_REQUESTED** [review]

File: results/test-results.xml (hunk 1 above)
Comment (jjtechuy) [reply results/test-results.xml]: You are right. Sorry for this.

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**